Vizzuality/heco-invest
# Add for-investors page

This PR adds the for-investors page

# BLOCKED Waiting for BE answer

## Testing instructions

Description

As an HeCo manager, I need curated information for investors so that they understand the value proposition of the platform and are encouraged to participate

See [“For investors” design](https://www.figma.com/proto/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?page-id=2%3A4&node-id=2%3A3784&viewport=4325%2C4007%2C0.4&scaling=scale-down&starting-point-node-id=2%3A2923&show-proto-sidebar=1) for reference

Acceptance criteria

There will be a page with specific information for investors

Note: “For investors”

It will be accessible from the home page (LET-219)

Note: this shouldn’t create a dependency. If this page is developed before the home page, this acceptance criterion should be removed and the link described in a different user story

The page will include the following information, complying with the structure and design defined at [“For investors” design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=2%3A3784)

Value proposition of the platform for the investor

Why should the investor invest in the Amazon

Explanation of categories that works as filters that links to the search

Explanation of priority landscapes that works as filters that links to the search

Carrossel with information of what the investor can do in the platform

TEST URL: https://heco-invest-frontend-oxrllhexz-vizzuality1.vercel.app/en/for-investors

## Tracking

[LET-711](https://vizzuality.atlassian.net/browse/LET-711)
